wdl2cwl/main_oop.py

```diff
@@ -562,8 +649,8 @@ def main() -> None:
         with open(args.output, "w") as result:
             result.write(str(Converter.load_wdl_tree(args.workflow)))
 
-    # Converter.load_wdl_tree("wdl2cwl/tests/wdl_files/bowtie_1.wdl")
-    # Converter.load_wdl_tree("wdl2cwl/tests/wdl_files/bcftools_stats.wdl")
+    Converter.load_wdl_tree("wdl2cwl/tests/wdl_files/bowtie_1.wdl")
+    Converter.load_wdl_tree("wdl2cwl/tests/wdl_files/bcftools_stats.wdl")
```

Comment:
Sorry about that. I leave them commented in case I need it during debugging. Trying to understand how objects are represented as compared to the file am currently working on.